index.html

```diff
@@ -0,0 +1,142 @@
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Microverse Trials</title>
+</head>
```

Comment:
* Nice job using the inline comments feature 👍 
* It is always better to include an explanation, a "why", for each requested change. For example:

This is a well-explained request: Why don't you use an article/section/aside/etc tag instead of a div, that would improve the HTML structure and make the page more accessible

This request lacks a "why" please make the necessary changes 🙏